### PR TITLE
Add nginx template for beanstalkd

### PR DESCRIPTION
**Pull request by MattiasGees**
This is a first version to add an template for beanstalkd console with nginx. I kept it simple and is advised to only run it behind fw/ssh tunnel.

### CODE REVIEW

File: manifests/params.pp [hunk 1]
```diff
@@ -25,4 +25,5 @@
   $webserver_port = '80'
   $vhostaddress = "beanstalkconsole.${::fqdn}"
   $web_username = 'beanstalkd'
+  $php_owner = 'www-data'
```
Comment (simonrondelez) [inline manifests/params.pp]: shouldn't this be the ${::customer::companyshortname}-data user?

**Review by simonrondelez — APPROVED** [review]
small remark

File: manifests/params.pp (hunk 1 above)
Comment (MattiasGees) [inline manifests/params.pp]: Isn't apache running under www-data?

Comment (simonrondelez) [inline manifests/params.pp]: ignore my comment then 👍

**Review by iuriaranda — APPROVED** [review]

**Pull request merged**